src/content/docs/apm/agents/java-agent/installation/java-install-overview.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: An overview of the Java APM installation process
```

Comment:
Not clear if you mean this doc was deleted and then restored. If that's the case, are there going to be any problems with redirects from when it was previously deleted?

Reply:
There are a couple of links that I'm updating too.